Library/Formula/bzr-fastimport.rb

```diff
@@ -0,0 +1,38 @@
+class BzrFastimport < Formula
+  homepage "https://launchpad.net/bzr-fastimport"
+  url "https://launchpad.net/bzr-fastimport/trunk/0.13.0/+download/bzr-fastimport-0.13.0.tar.gz"
+  sha1 "1f08dd85098d047c11c4b42e060038463d9c50ed"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "bazaar"
+
+  resource "python-fastimport" do
+    url "http://launchpad.net/python-fastimport/trunk/0.9.0/+download/python-fastimport-0.9.0.tar.gz"
+    sha1 "214cc2c01b6e6e9f137862c2ff487f335ec850ea"
+  end
+
+  def install
+    resource("python-fastimport").stage do
+      system "python", *Language::Python.setup_install_args(libexec/"vendor")
+    end
+
+    libexec.install Dir["*"]
+
+    # The plugin must be symlinked in bzrlib/plugins to work
+    ln_sf libexec, Formula["bazaar"].libexec/"bzrlib/plugins/fastimport"
+  end
+
+  def caveats; <<-EOS.undent
+    In order to use this plugin you must set your PYTHONPATH in your ~/.bashrc:
+
+      export PYTHONPATH="#{libexec}/vendor/lib/python2.7/site-packages:$PYTHONPATH"
```

Comment:
@mikemcquaid I’d love too but I don’t know how. Bazaar plugins unfortunately don’t work like Git subcommands: they’re python modules, not executable files.